### PR TITLE
Adding important note to solve #3493

### PR DESCRIPTION
**Pull request by j0rt3g4**
Adding Important note and link to solve #3493

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix Minor copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@JohanFreelancer9 Minor copy editing is needed for this PR. Thanks.

File: windows/security/identity-protection/hello-for-business/hello-hybrid-key-whfb-settings-dir-sync.md [hunk 1]
```diff
@@ -23,11 +23,14 @@ ms.date: 08/19/2018
...
 
 
+
 ## Directory Synchronization
 
 In hybrid deployments, users register the public portion of their Windows Hello for Business credential with Azure.  Azure AD Connect synchronizes the Windows Hello for Business public key to Active Directory.  
 
 ### Group Memberships for the Azure AD Connect Service Account
+>[!IMPORTANT]
+> If you already have a Windows Server 2016 domain controller in your domain, you can skip Configure Permissions for Key Synchronization. For more please check [Configure Hybrid Windows Hello for Business: Directory Synchronization](https://docs.microsoft.com/windows/security/identity-protection/hello-for-business/hello-hybrid-cert-whfb-settings-dir-sync)
```
Comment (nenonix) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-key-whfb-settings-dir-sync.md]:
```suggestion
> If you already have a Windows Server 2016 domain controller in your domain, you can skip **Configure Permissions for Key Synchronization**. For more detail see [Configure Hybrid Windows Hello for Business: Directory Synchronization](https://docs.microsoft.com/windows/security/identity-protection/hello-for-business/hello-hybrid-cert-whfb-settings-dir-sync).
```

**Comment by e0i** [comment]
@mapalko Would you review this PR based on issue #3493 ?

**Comment by e0i** [comment]
@mapalko Would you review this PR based on issue #3493 ?

**Comment by e0i** [comment]
@mapalko Would you be able to review and merge this PR? Thanks.

**Comment by e0i** [comment]
@Justinha Would you be able to review and merge this PR? Thanks.

**Comment by e0i** [comment]
@Justinha @DaniHalfin Would you be able to review and merge this PR? Thanks.

**Comment by DaniHalfin** [comment]
Looks good to me, but need owner to verify. @mapalko - makes sense to you ?

File: windows/security/identity-protection/hello-for-business/hello-hybrid-key-whfb-settings-dir-sync.md [hunk 2]
```diff
@@ -23,11 +23,14 @@ ms.date: 08/19/2018
 -   Key trust
 
 
+
```
Comment (illfated) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-key-whfb-settings-dir-sync.md]:
```suggestion
```

File: windows/security/identity-protection/hello-for-business/hello-hybrid-key-whfb-settings-dir-sync.md [hunk 3]
```diff
@@ -23,11 +23,14 @@ ms.date: 08/19/2018
 -   Key trust
```
Comment (illfated) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-key-whfb-settings-dir-sync.md]:
```suggestion
```

**Comment by e0i** [comment]
@mapalko @dulcemontemayor Would you confirm the updated PR? Thanks.

**Comment by mapalko** [comment]
@e0i @DaniHalfin updates look good.

**Comment by e0i** [comment]
Thank you.

**Pull request merged**